### PR TITLE
feat: Toolbox additional url search parameter

### PR DESCRIPTION
**Pull request by tbwiss**
# Description

> **Note**
> 
> Please provide a description of the work completed in this PR below

For future Graph Academy courses it's thought to use the GraphQL Toolbox. To make this a smooth and enjoyable experience, the GraphQL Toolbox needs to pre-fill and automatically connect to the **temporary** sandbox Neo4j database of the Graph Academy course(s).

This can be solved by extending the existing `connectURL` to include a password value, see the example below.
If a complete `connectURL` URL parameter is provided (with username, pw, and database uri), the GraphQL Toolbox will attempt one "auto" login on startup/load of the web page.
Example: `http://<url-for-graphql-toolbox>?connectURL=bolt%2Bs://neo4j:mypassword@xxxxxx.databases.neo4j.io`

-> It goes without saying that the password is exposed when using this approach, therefore it shall only be used for _test_ or _temporary_ (sandbox) Neo4j databases.

## Complexity

> **Note**
>
> Please provide an estimated complexity of this PR of either Low, Medium or High

Complexity: Medium

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Made some changes to the GraphQL Toolbox? Or made some changes to the core library perhaps?

Check out your changes at https://neo4j-graphql-toolbox-stage-2702.surge.sh

Use for instance a local dbms hosted on [Neo4j Desktop](https://neo4j.com/download/) or a cloud-hosted [Aura Free](https://neo4j.com/cloud/aura-free/) dbms.

File: packages/graphql-toolbox/README.md [hunk 1]
```diff
@@ -67,6 +68,12 @@ This will pre-fill the login window input fields `Username` with `testuser` and
 
 Note: The plus symbol (+) in the connectURL needs to be URL encoded to become `%2B`. For example `bolt+s` becomes `bolt%2Bs`.
 
+The URL parameter `pw` is additionally available to provide the database password.
```
Comment (oskarhane) [inline packages/graphql-toolbox/README.md:71]: Any particular reason for the password being a separate parameter?
https://datatracker.ietf.org/doc/html/rfc1738#section-3.1

Comment (tbwiss) [reply packages/graphql-toolbox/README.md:71]: Mm missed this one. Will refactor

Comment (oskarhane) [reply packages/graphql-toolbox/README.md]: Cool.
Don't forget to update the description of this PR as well after the changes 🚋

**Comment by tbwiss** [comment]
We do not go forward with this approach, instead, we wait for an integration using SSO. Closing this.

**Pull request closed without merging**